src/Test.UnitTests.Sarif.Driver/Sdk/AnalyzeCommandBaseTests.cs

```diff
@@ -36,6 +36,44 @@ public AnalyzeCommandBaseTests(ITestOutputHelper output)
             Output.WriteLine($"The seed that will be used is: {TestRule.s_seed}");
         }
 
+        [Fact]
+        public void AnalyzeCommandBase_ScanWithNoValidTargetsAndOneOrMoreSkippedFilesEmitsSkippedFilesWarning()
```

Comment:
>ScanWithNoValidTargetsAndOneOrMoreSkippedFilesEmitsSkippedFilesWarning [](http://example.com/codeflow?start=39&length=70)

throw a much less precise, but easier to read version: `ScanWithOnlyOversizedFilesEmitsFilesSkippedWarning`

it is not precise but it meets what is actually tested in this test.

Reply:
Without look into code details my understanding the purpose of the change at first sight was wrong. :)

The test names looks great!